com.unity.render-pipelines.high-definition/Editor/ShaderGraph/HDTarget.cs

```diff
@@ -212,6 +213,10 @@ static class CoreFieldDependencies
             new FieldDependency(StructFields.SurfaceDescriptionInputs.FaceSign,                      HDStructFields.FragInputs.IsFrontFace),
 
             new FieldDependency(HDFields.DepthOffset,                                                HDStructFields.FragInputs.positionRWS),
+
+            new FieldDependency(StructFields.SurfaceDescriptionInputs.BoneWeights,                   StructFields.SurfaceDescriptionInputs.BoneIndices),
+            new FieldDependency(StructFields.SurfaceDescriptionInputs.BoneIndices,                   StructFields.SurfaceDescriptionInputs.BoneWeights),
+            new FieldDependency(StructFields.SurfaceDescriptionInputs.VertexID,                      StructFields.SurfaceDescriptionInputs.VertexID),
```

Comment:
Ideally we would not have these be part of surface descriptions? However, they are required to have the nodes work in subgraphs...